web/client/components/mapcontrols/annotations/AnnotationsEditor.jsx

```diff
@@ -262,6 +290,18 @@ class AnnotationsEditor extends React.Component {
         );
     }
 
+    zoom = () => {
+        if (this.props.feature.geometry.type === 'MultiPoint') {
+            const extent = this.props.feature.geometry.coordinates.reduce((previous, current) => {
```

Comment:
This as many other should be implemented once by implementing a zoom to feature action. Anyway you can use `@turf/bbox` instead of reinventing bbox calculation of multipoint.

Reply:
@mbarto can you do this improvement or should we merge this PR?

Reply:
I can do it, you just have to wait a little more :-)